index.js

```diff
@@ -1,8 +1,17 @@
 module.exports = robot => {
   robot.on('issues.opened', async context => {
-    const options = context.repo({path: '.github/ISSUE_REPLY_TEMPLATE.md'});
-    const res = await context.github.repos.getContent(options);
-    const template = new Buffer(res.data.content, 'base64').toString();
+    let template;
+    try {
+      const options = context.repo({path: '.github/ISSUE_REPLY_TEMPLATE.md'});
+      const res = await context.github.repos.getContent(options);
+      template = new Buffer(res.data.content, 'base64').toString();
+    } catch (err) {
+      if (err.code === 404) {
+        template = 'We couldn\'t find an `ISSUE_REPLY_TEMPLATE` in your `.github` folder!';
+      } else {
+        throw err;
+      }
+    }
```

Comment:
@bkeepers I totally agree, it's been something I've been struggling with writing my own. I also considered the opening a PR/Issue method and I do like it better than confusing the original person. 

> The challenge is what to do when you install an app on an entire organization.

Do some apps act on an organizational level? It seems like if they do they must handle this already somehow?

Also, imbedding that semi-complex logic into plugins will make their code base less readable, considering this plugin was originally literally 8 lines of code. 

Would this be something we could consider building into probot directly? As in every plugin on first installation would open a Pull Request with plugin specific changes and content. Building that logic into every plugin seems potentially tedious, but I'm not sure how possible it would be to build into probot itself as a method plugins could use?

On a semi-relevant note, I would love to talk through best practices about config files. One thing that I have found is that while writing a series of simpler plugins, having multiple running on a repo can really make your `.github/` folder hideous ([right now my test account has 6 different files in that folder](https://github.com/hiimbex/testing-things/tree/master/.github) 😓). Theoretically multiple plugins could all draw data from a `.github/config.yml` file so long as all the fields contain different names; however, with increasing amounts of plugins that gets tricky. But if you'd like to install 5 plugins having a `.github/ISSUE_REPLY_TEMPLATE.md` equivalent for each doesn't seem like the best way to go..